.github/workflows/verify.yml

```diff
@@ -44,3 +44,8 @@ jobs:
 
     - name: Codecov report
       run: bash <(curl -s https://codecov.io/bash)
+
+    - name: Build docs
+      run: make site-build
+name: Verify docs build
+on: [pull_request]
```

Comment:
Are these needed here at the end ? Aren't these then (unintentioanlly?) changing the behavior of the workflow to be triggered on just for PRs instead of the originally inteded push or PR (vs. the on:-line  in the beginning of the file) ?

Reply:
Waat have I done?! Copy-paste remnants that really should not be there 

I was just too quick on this one, good that you reviewed it 😌  Fixed now.